packages/design-system/src/components/OcBreadcrumb/OcBreadcrumb.vue

```diff
@@ -270,6 +304,21 @@ export default defineComponent({
       return props.items.length - 1 === index ? 'page' : null
     }
 
+    const dropItemStyling = (id, index, leaving, event) => {
+      if (index === unref(displayItems).length - 1) {
```

Comment:
Since you use the same early return here as in the `dropItemEvent` function: it would make sense to extract the checks into a `isDropAllowed` helper.